### PR TITLE
Ds 426: datepicker styles fixes

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/components/DatePicker/index.tsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ const DatePicker: FC<DatePickerProps> = ({
 
   const handleDisplayDateOptions = () => {
     if (!isMonthPickerActive && !isYearPickerActive) {
-      setIsMonthPickerActive(true);
+      setIsYearPickerActive(true);
     }
```
Comment (pauzim) [inline packages/design-system/src/components/DatePicker/index.tsx:147]: Not sure about that - in the first scenario by clicking once on month the user can choose the month from the month list, and with another click year picker appears,  now there's a year picker only

**Review by pauzim — COMMENTED** [review]
In range variant styles don't fit:
Screenshot 2023-05-10 at 15 37 05

https://www.figma.com/file/xy6otn2JWHNdF70Tuq0UbS/TACO-Design-System-%5BDRAFT%5D?type=design&node-id=4228-71144&t=3L4HGuuhI8Y99MBA-0

**Comment by zuzannanap** [comment]
> In range variant styles don't fit: Screenshot 2023-05-10 at 15 37 05
> 
> https://www.figma.com/file/xy6otn2JWHNdF70Tuq0UbS/TACO-Design-System-%5BDRAFT%5D?type=design&node-id=4228-71144&t=3L4HGuuhI8Y99MBA-0

right, i didn't noticed, fixed!

File: packages/design-system/src/components/DatePicker/index.tsx (hunk 1 above)
Comment (zuzannanap) [reply packages/design-system/src/components/DatePicker/index.tsx:147]: before when you clicked the month-year button it showed month to choose and didn't ask for a year
now when you click the button you choose the year and when you click selected year, it shows months to select one of them - i think we can discuss about it with others

Comment (pauzim) [reply packages/design-system/src/components/DatePicker/index.tsx:147]: When you were on the view "month to choose" and click again month-year button the list of the years appears. Not sure, but I think this mechanism was discussed with Alex

**Review by pauzim — APPROVED** [review]
Now styles looks gooood :) please ask Alex how month-year button should behave

**Pull request merged**